tests/measurements/test_expval.py

```diff
@@ -188,7 +187,7 @@ def circuit():
 
         res = circuit()
         expected = [0.5, 0.5] if isinstance(shots, list) else 0.5
-        assert np.allclose(res, expected, atol=0.02, rtol=0.02)
+        assert np.allclose(res, expected, atol=0.02, rtol=0.035)
```

Comment:
```suggestion
        assert np.allclose(res, expected, atol=0.02, rtol=0.04)
```
`expected=0.5` so `atol` is already 4%. It's fine to just lift `rtol` to same.